src/newrelic/monitor_script.js

```diff
@@ -16,67 +16,73 @@
 const assert = require('assert');
 
 // $http -> https://github.com/request/request
-$http.get('$$$URL$$$',
-  // callback
-  (err, response, body) => {
-    const status = JSON.parse(body);
-    Object.keys(status).forEach((v) => {
-      if (['status', 'error', 'process', 'version', 'response_time'].indexOf(v) === -1) {
-        $util.insights.set(v, parseInt(status[v], 10));
-      }
-    });
-    $util.insights.set('status', status.status);
-    ['x-openwhisk-activation-id', 'x-request-id', 'x-version'].forEach((h) => {
-      $util.insights.set(h, response.headers[h]);
-    });
-    if (status.error) {
-      $util.insights.set('errorStatus', status.error.statuscode);
-      $util.insights.set('errorURL', status.error.url);
+$http.get({
+  url: '$$$URL$$$',
+  headers: {
+    // make sure activation records gets persisted
+    'X-OW-EXTRA-LOGGING': 'on',
+  },
+},
```

Comment:
This is the actual change, the rest of the file is just whitespace adjustments

Reply:
why change indention?

Reply:
Because eslint didn't like it anymore after I turned the changed argument from string to an object 😄